Description: Make OpenSSL header detection more reliable
 <openssl/opensslconf.h> is architecture-dependent and so may reside in a
 different location on multiarch-capable systems.  Check for
 <openssl/opensslv.h> instead.
Author: Colin Watson <cjwatson@ubuntu.com>
Forwarded: no
Last-Update: 2012-12-20

--- a/m4/ntp_openssl.m4
+++ b/m4/ntp_openssl.m4
@@ -86,10 +86,10 @@
 case "$ans" in
  no)
     ;;
- *) # look for openssl/opensslconf.h:
+ *) # look for openssl/opensslv.h:
     for i in $ans no
     do
-	test -f $i/openssl/opensslconf.h && break
+	test -f $i/openssl/opensslv.h && break
     done
     case "$i" in
      no)
